modules/_labs/menu/react/stories/stories_visualTesting.tsx

```diff
@@ -47,14 +56,14 @@ const createMenuItems = (hasIcons?: boolean, isFocused?: boolean): StoryMenuItem
         ''
       ) : (
         <em>
-          Fourth Item (<b>with markup</b>)
+          Sixth Item (<b>with markup</b>)
         </em>
       ),
-      icon: undefined,
+      icon: hasIcons ? userIcon : undefined,
```

Comment:
This was added to be consistent with the other story. This was previously rendering an empty item in the menu and was confusing.